packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -68,22 +72,21 @@ export default createConnector({
     const {results} = searchResults;
     const {attributeName, showMore, limitMin, limitMax} = props;
     const limit = showMore ? limitMax : limitMin;
-
     const canRefine =
       Boolean(results) &&
       Boolean(results.getFacetByName(attributeName));
 
     const isFromSearch = Boolean(searchForFacetValuesResults
       && searchForFacetValuesResults[attributeName]
       && searchForFacetValuesResults.query !== '');
-    const searchForFacetValues = props.searchForFacetValues ? this.searchForFacetValues : undefined;
+    const withSearchBox = props.withSearchBox || props.searchForFacetValues;
```

Comment:
Can we add a if (process.env.NODE_ENV === 'development') console.warn:

```text
react-instantsearch: `searchForFacetValues` has been renamed to `withSearchBox`, this will break in the next major version.
```